build/configs/imxrt1050-evk/tc/defconfig

```diff
@@ -937,12 +987,15 @@ CONFIG_TC_FS_PROCFS=y
 # CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC is not set
 # CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC is not set
 # CONFIG_EXAMPLES_TESTCASE_SECURITY_FRAMEWORK_UTC is not set
+# CONFIG_LIBCXX_UTC is not set
```

Comment:
Isn't it the main topic of this commit? This is not enabled.

Reply:
I can enable these test cases by default. I thought like some other test cases, can be enabled during specific tests, might increase binary size.